src/middlewared/middlewared/plugins/chart_releases_linux/secrets_management.py

```diff
@@ -14,6 +15,8 @@
 
 class ChartReleaseService(Service):
 
+    LOGGED_SECRET = set()
```

Comment:
How large can this grow? I don't like idea of storing secret keys in memory for life of process while it's running.

Reply:
For most users it does not happen and for the user in question it was 2 secrets..
I don't expect it to be a widespread problem

Reply:
Okay, I trust you 😄